src/blocks/music.rs

```diff
@@ -131,16 +132,18 @@ struct OwnerChange {
 pub async fn run(config: toml::Value, mut api: CommonApi) -> Result<()> {
     let config = MusicConfig::deserialize(config).config_error()?;
     let dbus_conn = new_dbus_connection().await?;
-    let mut widget = api
-        .new_widget()
-        .with_icon("music")?
-        .with_format(config.format.with_default("$combo.rot-str() $play|")?);
+    let mut widget = api.new_widget().with_format(
+        config
+            .format
+            .with_default(" $icon {$combo.rot-str() $play|} ")?,
```

Comment:
This is out of sync with the docs

Reply:
Now it has double spaces...

Reply:
```suggestion
            .with_default(" $icon{ $combo.rot-str() $play|} ")?,
```

or

```suggestion
            .with_default(" $icon {$combo.rot-str() $play |}")?,
```

Reply:
I've applied the suggestion and fix rest of format docs :|

https://github.com/greshake/i3status-rust/pull/1618/commits/ed585d04772be461639b62769352deb6614b3209